packages/shared/src/modules/users/libs/validation-schemas/user-sign-up.validation-schema.ts

```diff
@@ -7,21 +7,25 @@ import {
 } from "../enums/enums.js";
 
 type UserSignUpRequestValidationDto = {
+	confirmPassword: z.ZodString;
```

Comment:
You should remove confirm password field from validation schema

Reply:
But the requirement say we should be shown in real time validation

Reply:
Confirm password field should not be sent to backend. But if you will add it to validation schema it will be sent on backend. Check #168 pull request and comments under it. Update error message in packages/shared/src/modules/auth/libs/enums/confirm-password-custom-validation.enum.ts

Reply:
Thanks for your explanation,  i try to remove this field